### PR TITLE
Change on Unit enums

### PR DESCRIPTION
**Pull request by AbbosB12**

### CODE REVIEW

File: src/main/java/com/example/accountingapp/entity/Category.java [hunk 1]
```diff
@@ -18,8 +18,6 @@
 public class Category extends BaseEntity{
 
     private String description;
-    @Enumerated(EnumType.STRING)
-    private Unit unit;
```
Comment (MehmetKaldirim) [inline src/main/java/com/example/accountingapp/entity/Category.java:22]: As far as I saw , there is no unit field in the category entity, it should be in the product entity... u re just charge of creating enums and it s well done...

**Review by MehmetKaldirim — APPROVED** [review]

**Review by mgulmira — APPROVED** [review]
looks good please add data to category in sql file. Approving this PR so you could create another for sql

**Comment by mgulmira** [comment]
looks like my recent merge to master create conflicts. I uncommented Company entity in Category. Please rebase and manually resolve conflicts and push again

File: src/main/java/com/example/accountingapp/enums/Unit.java [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+@Getter
 public enum Unit {
-    PIECES("Pieces");
+    LBS("Libre"),
+    GALLON("Gallon"),
+    PCS("Pieces"),
+    KG("Kilogram"),
+    METER("Meter"),
+    INCH("inch"),
+    FEET("Feet");
```
Comment (VitalySlabada) [inline src/main/java/com/example/accountingapp/enums/Unit.java]: I know you are using input from Cihat but what is libre :) ?  pounds ? 
"inch"  should start with Capital or everything should be lower cas 

frankly I would change to standard  acronyms (kg, in , lbs, m, ft, pcs etc ) I think it looks more professional and takes less space
just my opinion - feel free to disagree :)

**Pull request merged**